packaging/debian-sid/rules

```diff
@@ -61,8 +61,10 @@ endif
 
 # check if we need to include the testkeys in the binary
 TAGS=nosecboot
+SNAP_TAGS=nosecboot nomanagers
```

Comment:
Maybe a comment here (and elsewhere) that `TAGS` are the go build tags for all binaries. And that `SNAP_TAGS` is only for the "snap" binary?